src/content/en/2022/sustainability.md

```diff
@@ -1,22 +1,814 @@
 ---
-#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/
 title: Sustainability
-#TODO - Review and update chapter description
-description: Sustainability chapter of the 2022 Web Almanac covering ...
-authors: [ldevernay, gerrymcgovernireland, tpgreenwood, timfrick]
-reviewers: [mrchrisadams, cqueern, Djohn12, hanopcan]
+description: Sustainability chapter of the 2022 Web Almanac covering environmental impacts of web pages, where they come from and how to reduce them
+authors: [ldevernay, gerrymcgovernireland, timfrick]
+reviewers: [mrchrisadams, cqueern, Djohn12]
 analysts: [fershad, camcash17, 4upz]
```

Comment:
@ldevernay can you please check all the contributors are listed in each section (in descending order of contribution) and that we haven't forgotten anyone?

Reply:
Already checked that, we're OK!